### ISSUE TITLE
When will the Chinese version be launched

### ISSUE THREAD

**Issue by lyx582** [kingToolbox/WindTerm#211]
When will the Chinese version be launched

**Comment by kingToolbox** [comment]
Very soon, It is expected to support multiple languages in early October. But at the beginning there will only be a machine-translated version, and the human-translated version depends on the contributor.

**Comment by kingToolbox** [comment]
@ranbest
Yes, this is what I will do. In addition, in order to save memory during development, many strings are spliced, and now they need to be modified to complete strings to facilitate translation. Some user interfaces also need to be resized to fit the display in different languages.

The extraction of strings has begun.

**Comment by OIdDriver** [comment]
pre3已支持，美滋滋。

**Comment by kingToolbox** [comment]
The new [WindTerm_2.2.0_Prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.2-prerelease) version has been released and a better Chinese interface has been supported, please download and check it, thank you.
